suds/transport/http.py

```diff
@@ -74,7 +74,7 @@ def send(self, request):
             self.proxy = self.options.proxy
             request.headers.update(u2request.headers)
             log.debug('sending:\n%s', request)
-            fp = self.u2open(u2request)
+            fp = self.u2open(u2request, timeout)
```

Comment:
how confident are we that calling:

     u2open(u2request, None)

is the same as:

    u2open(u2request)